modules/cas_cache/context.c

```diff
@@ -66,6 +67,11 @@ void *_cas_alloc_page_rpool(void *allocator_ctx, int cpu)
 
 	_cas_page_set_priv(page);
 	_cas_page_set_cpu(page, cpu);
+
+	/* As it isn't an actuall allocation but only a rpool preparation
+	   kmemleak shouldn't track the allocated memory yet. */
+	kmemleak_free(page_address(page));
+
```

Comment:
I think this is not correct, since you mentioned page allocations are not tracked by kmemleak

Reply:
+1, normally it will not be very visible, but it will generate warning on debug kernel.